homeassistant/components/thermostat/zwave.py

```diff
@@ -52,6 +52,10 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                 _LOGGER.debug("Remotec ZXT-120 Zwave Thermostat, ignoring")
                 return
 
+    if value.command_class != COMMAND_CLASS_SENSOR_MULTILEVEL and \
+       value.command_class != COMMAND_CLASS_THERMOSTAT_SETPOINT:
+        return
```

Comment:
No, This is inverted logic. We use the same here:
https://github.com/turbokongen/home-assistant/blob/zwave_fixes/homeassistant/components/garage_door/zwave.py#L28